drivers/hmis/app/models/hmis/hud/custom_client_contact_point.rb

```diff
@@ -56,7 +56,7 @@ def self.system_values
   end
 
   def equal_for_merge?(other)
-    columns = [:system, :use, :value]
+    columns = [:system, :value]
```

Comment:
This changes logic for client merge. We'll no longer keep 2 phones/emails if they have the same value but differing "use" (home/work/temp)